lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+    "iconURL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAbFBMVEX////gSj+qODDgSj/o5tiwr6ThVEniZ1zn08XOzMCqODDmtajl49bY18nkjoLdST7Lyr2/VUuvoJblopXWRzzFxLjT0cW6aV/mv7LjhXjPRDq0fXPCQDfMS0Hg3tHe3M60sqfl49W7ua6vkYfYzKRYAAAAA3RSTlMA19csuNBxAAAAaElEQVR4Xp3IRw7CMBQFwJj3m51e6f3+dyTYoGSJmOVkP3Bq9GHq5gAkT5ELDjGA6h0VcAox9oCaKeptSEE9IILhyN+gnQDnJy9Bpm3H6yj9yOvwJdG0RNFcaHa9pXBoCoruj7DJ/vACm58E7AMvk10AAAAASUVORK5CYII=",
+    "name": "Gmail",
+    "origin": "https://mail.google.com",
+    "pageSize": {
+      "share": {
+        "height": 500,
+        "width": 500
+      }
+    },
+    "shareURL": "https://mail.google.com/mail/?view=cm&ui=2&tf=0&fs=1&body=%{body}&su=%{title}",
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/gmail.html"
+  },
+  "social.manifest.www-linkedin.com": {
+    "author": "LinkedIn Corporation",
+    "description": "Manage your professional identity. Build and engage with your professional network. Access knowledge, insights and opportunities.",
+    "icon32URL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAZCAYAAAAmNZ4aAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAABJFJREFUeNqsVl1oHFUU/u7M7OxfyuanadNtlGz8KYl5kASsD1oEqfgiJvimfRBUCPgkFd98UF/6VtASEMQfahERCiJCfahIKVYakuJPbWsTo92ENSaN3U2ymZ07M9dzZnY3O7OTPkRnuXtnzrn3fOfvnnPFniPPH3/m5dfe2pfLZl3Xw//16LqOxduV9UufvffqlU9OnO49cDDEFxMfXraPHupLSMelT8WkOiv6jrvw4tcx+IXrxfmT4w+N5HtyViuwYSYSiVLFgiRrBa2XLvx3XQNM/vOFtAqMUwKx63RNIJntyAkznSJCGLhsSaxu2XA9BdtVuKczjbF9OZTWLfxcWkfCB98dsCBLKpb0aFLRUBgb0sE/BE58FLozeOOJAjpM3WdO/XAL39xYQdLQdhVnQb8tCmEMLoyq7aJsOaSZg4mR/U1Qfo4UuvHFTyUknd0BM9yWdJVoeqcFmDXaqDmokuXX/t4IMW+sbIJDkUnouwLmMxKclBhgixjrBOqRfl9dX8beDhNP3teDm7erOHlxAbbnwZVqZ+FK+S4VIo5HmQ0/1O3ANccja4OjRF7B2+dv4sR3c5C0aw+5PWsaqJJXtmy3uSlNdEn7GC1L764XyNDpO5XQmjlG4pDUhIoFdshim4TYNBe6Mnjn6CHaF1hx7rcVfH1tGWP9nXhxrL+56dT3f6A/l8Irj9yLQUpIScC/Lm/gg+kiZpbKyFJoWAZH19ghSgZbaTsKm7aHrpSJFx7ON5lLZQufziyiryMZoqcoy58d3g9D2zZlqLcDTz/Yi4nTM5gulpEmy9kTptDjDIbmEbBDFrtsteOGmDU6YuwvW4bpz430hUAbD7v9+OOD4CrIMjkcnhefH5qi5FDkZsUBiSxiHsgbKmbvmdklvPT5j/h4ejFEHz2YQ2/GDHKAi1JNQkrZ7mquWJJy2vF4hBGYB85qL9w8zlFROXZm1n//iOJ6/94sHit0+d+daQOdKQPLVPkMTYNUDlnvtFvMYDaNGg0nYhofMdY6qtC3c6v+Ic0kDd9Ts5RQTUsoBIJGzQ3kun4PiTnHLruaDfItiwA36BFXc8Fh//sK0WxFcoPlcGwdQbNQKi65DDbSt4xevKjFKp5eT4AmP66oNPisWywwgwoVCFKRYq7qCrXTw8AqItqr81Uw4noEtNbGFlVe1aXE0kP7oqchZL3SY6qI0SYl0tb8vhqJkmisr4/W3AnzFDKZDH5fmHfe//LsA47rjk5OTp7P5/Orhgoh7wAgYvpdIwRKtbGFqCNzrU6msLS4qE9NTR0mxqOjo6Ol8fHxC9sW0/3ol78qOPzuxSborTtbAJXLS3+uhegLa1WSaAQK0HyKuthZ6tsNUQvU2aBp9Th7wjB0t6en+0rfgbwcHh6eC1y9rSY26TJweX615aqo8cFEuSrb6XogGHRmiwRUXGnp5RzT+k2IslqjKUFfV4eGhq4ODAzUz7vGNUQZvhBeHG36d6M3Zl34Hou7guTSZiWtS0v5lwLXr2KmaUJ7KlV6XbOtO7Gd/D89AglprR0b1N5MG8LigqJp21eofwUYABnolxra9d85AAAAAElFTkSuQmCC",
```

Comment:
Is `homepageURL` a required field?

```
social.manifest.www-linkedin.com:
  child "homepageURL" fails because ["homepageURL" is required].
  child "version" fails because ["version" is required]
```